corehq/apps/importer/const.py

```diff
@@ -1,2 +1,11 @@
 class LookupErrors:
     NotFound, MultipleResults = range(2)
+
+
+class ImportErrors:
+    InvalidOwnerName = 'Invalid Owner Name'
+    InvalidOwnerId = 'Invalid Owner ID'
+    InvalidParentId = 'Invalid Parent ID'
+    InvalidDate = 'Invalid Date'
+    BlankExternalId = 'Blank External ID'
+    CaseGeneration = 'Case Generation Error'
```

Comment:
> It's translated here: ...

I was looking for that, but couldn't find it. Will the translation engine know how to find these strings when they're collected and sent to the translators? My understanding is that a kind of static analysis is done on the code to find strings to translate, and it seems unlikely that these strings would be found by that system. @twymer can you verify that?